packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, FocusableRefValue, StyleProps} from '@react-types/shared';
+import {classNames, createFocusableRef, SlotProvider, useStyleProps} from '@react-spectrum/utils';
+import {DropOptions, mergeProps} from 'react-aria';
+import {filterDOMProps} from '@react-aria/utils';
+import {DropZone as RACDropZone} from 'react-aria-components';
+import React, {ReactNode, Ref, useImperativeHandle, useRef} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/dropzone/vars.css';
+export interface SpectrumDropZoneProps extends Omit<DropOptions, 'getDropOperationForPoint' | 'ref' | 'hasDropButton'>, DOMProps, StyleProps, AriaLabelingProps {
```

Comment:
I think we may want to extend RAC's `DropZoneProps` (with `'children'` omitted) here instead of `DropOptions`? Or at least see if that simplifies this.